recipes/loky/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "loky" %}
+{% set version = "2.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cf17e866101bdb59f6d371fda916a680712af0879f2f536e5d880fcc128e9edf
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cloudpickle
+
+test:
+  imports:
+    - loky
+
+about:
+  home: https://github.com/tomMoral/loky
+  license: BSD
+  # No license yet
+  # license_file:
```

Comment:
Currently there isn't a license file. Raised issue ( https://github.com/tomMoral/loky/issues/136 ) to add one and get it packaged.